CodingTracker.kkvzx/Views/helpers/UserInputHandler.cs

```diff
@@ -0,0 +1,31 @@
+using Spectre.Console;
+
+namespace CodingTracker.Views;
+
+public static class UserInputHandler
+{
+    private static int GetId(TextPrompt<string> prompt)
+    {
+        var userInput = AnsiConsole.Prompt(prompt);
+        int id;
+
+        while (!int.TryParse(userInput, out id))
+        {
+            userInput = AnsiConsole.Prompt(new TextPrompt<string>("[red]Invalid format, please try again![/]"));
```

Comment:
You can simplify asking user for input by using `AnsiConsole.Ask<T>()`
```suggestion
            userInput = AnsiConsole.Ask<string>("[red]Invalid format, please try again[/]");
```